extensions/stackdriver/common/utils.cc

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "extensions/stackdriver/common/utils.h"
+#include "extensions/stackdriver/common/constants.h"
+
+namespace Extensions {
+namespace Stackdriver {
+namespace Common {
+
+using google::api::MonitoredResource;
+
+MonitoredResource getMonitoredResource(
```

Comment:
The common pattern is to pass a pointer/reference to the monitored resource so it's not copied on the method return, e.g. `updatedMonitoredResource(..., MonitoredResource& out)`

Reply:
Done.